handbook/engineering/observability/query.md

```diff
@@ -0,0 +1,15 @@
+# Manual searching developer guide
+
+> **Note:** Looking for _how to monitor Sourcegraph?_ See the [observability documentation](https://docs.sourcegraph.com/admin/observability).
+
+This document gives a quick overview of how to manually search for data that isn't currently represented or displayed in [Grafana dashboards](https://docs.sourcegraph.com/admin/observability/metrics). Note that, in most cases, writing manual queries indicates that you should be [adding monitoring](https://about.sourcegraph.com/handbook/engineering/observability/monitoring#adding-monitoring) so that the next person who needs to find that data doesn't need to repeat your work. 
```

Comment:
```suggestion
This document gives a quick overview of how to manually search for data that isn't currently represented or displayed in [Grafana dashboards](https://docs.sourcegraph.com/admin/observability/metrics). If the data you are querying is to monitor the health of Sourcegraph, you should be [adding monitoring](https://about.sourcegraph.com/handbook/engineering/observability/monitoring#adding-monitoring) instead so that the next person and site admins/customers alike don't need to repeat your work. 
```

Rephrasing slightly because we only want data in our monitoring stack if it can be alerted on, non-alertable (read: non-actionable) data can be queried ad-hoc (so this doc is great) but is usually a symptom of us not having a good way to alert on that data for some reason.

Reply:
Makes sense, thanks for the feedback!